clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -100,7 +90,8 @@ class AzureResourceManagerClient extends AzureBaseClient {
     ResourceGroupListParameters parameters = new ResourceGroupListParameters()
     parameters.setTagName("filter")
     parameters.setTagValue(applicationName)
-    return this.getResourceManagementClient(creds).getResourceGroupsOperations().list(parameters).resourceGroups
+
+    this.getResourceManagementClient(creds).getResourceGroupsOperations().list(parameters).resourceGroups
   }
 
   boolean resourceGroupExists(AzureCredentials creds, String resourceGroupName) {
```

Comment:
but you are still not using the resourceGroupName parameter and I don't expect the return types match correctly.